packages/suite/src/views/wallet/anonymize/components/CoinjoinSessionDetail.tsx

```diff
@@ -15,7 +15,7 @@ interface CoinjoinSessionDetailProps {
     hours: [number, number];
     maxFee: number;
     maxRounds: number;
-    skipRounds: [number, number] | null;
+    skipRounds?: [number, number] | null;
```

Comment:
```suggestion
    skipRounds?: [number, number];
```
Usable without `null`?

Reply:
I want it to be nullable explicitly, but the `?` could be removed: 93673b2e59367ed13545dc2e5117e9c1433d505d